### ISSUE TITLE
Mesh Without Material

### ISSUE THREAD

**Issue by JrIndie** [OGRECave/blender2ogre#15]
Hello, I was wondering if Blender2Ogre could export a mesh without a material attached to it.  For example, we want to have *.mesh with no predefined material so we can assign our own.

Thanks you!

**Comment by paroj** [comment]
https://github.com/OGRECave/blender2ogre#external-ogre-materials ?

**Comment by JrIndie** [comment]
Hello paroj,

Thank you for the response.  I am sorry I did not see this.  But I also want to clarify a few things ...

We are new to Ogre and just want to get the basics done correctly.  After getting meshes loaded and with their material and textures, I can only assume the internal workings of the function automatically load and apply materials/textures that are predefined in the *.mesh itself.

If this is correct, I would like to know of a way that does not do this, so that in our editors we can assign our own materials, and textures (basically like HLMS PBR).

Thank you for your time.

**Comment by paroj** [comment]
the mesh file merely stores a material name that is resolved on loading from the supplied *.material and the programmatically created materials.
So if you would like to use a material from the HLMS sample (1.10) you would call the material `extern.PBS_X_Y_Z` and make sure that the according material is create before the mesh is loaded.

**Comment by JrIndie** [comment]
Okay!  So there is no way we can leave this section blank, and supply the Material in our own tools?  If so, I assume we can use some basic Generic Material, and then we can change that later, correct?

Thanks!

**Comment by paroj** [comment]
yes, this would be the most straighforward approach. Note that Ogre requires that *some* material is specified - otherwise we would not know how to render the mesh.

**Comment by JrIndie** [comment]
paroj, a bit confused, inside the samples media folder, there is no such material called PBS_X_Y_Z, unless you mean for us to create our own basic material for PBS?

**Comment by JrIndie** [comment]
Not sure if it matters however when I do:   extern.PBS_GENERIC.material.  It still exports a material, and doing the following:

Ogre::MaterialPtr materialPtr = subEnt->getMaterial();
std::cout << "subEnt Material Name: " << subEnt->getMaterialName() << std::endl;

Outputs the following:  "subEnt Material Name: BaseWhite"

Not sure whats going on...
